README.md

```diff
@@ -0,0 +1,240 @@
+
+```
+packer {
+  required_plugins {
+    image-builder = {
+      source = "github.com/lzap/image-builder"
+      version = ">= 0.0.2"
+    }
+  }
+}
+
+source "image-builder" "example" {
+    build_host {
+        hostname = "buildhost.example.com"
+        username = "builder"
+    }
+
+    container_repository = "quay.io/centos-bootc/centos-bootc:stream9"
+
+    blueprint = <<BLUEPRINT
+[[customizations.user]]
+name = "user"
+password = "changeme"
+groups = ["wheel"]
+BLUEPRINT
+
+    image_type = "raw"
+}
+
+build {
+    sources = [ "source.image-builder.example" ]
+}
+```
+
+See [osbuild blueprint reference](https://osbuild.org/docs/user-guide/blueprint-reference/) for more info about blueprint format.
+
+Perform the build via:
+
+      packer init template.pkr.hcl
+      packer build template.pkr.hcl
+
+The image builder plugin will print last several lines from the image builder output as an artifact. To see more detailed output:
+
+      PACKER_LOG=1 packer build template.pkr.hcl
+
+### Config mapping
+
+For more info: https://github.com/osbuild/bootc-image-builder
+
+* **build_host.hostname** - IP or hostname with optional SSH port (required)
+* **build_host.username** - either root or username with sudo permissions (required)
+* **build_host.password** - SSH password when SSH keys are not available
+* **container_repository** - maps to container repository argument
+* **blueprint** - maps to `--blueprint`
+* **image_type** - maps to `--type`
+* **rootfs** - maps to `--rootfs`
+* **aws_upload.ami_name** - maps to AMI cloud uploader configuration
+* **aws_upload.s3_bucket** - maps to AMI cloud uploader configuration
+* **aws_upload.region** - maps to AMI cloud uploader configuration
+* **aws_upload.access_key_id** - maps to AMI cloud uploader configuration
+* **aws_upload.secret_access_key** - maps to AMI cloud uploader configuration
+
+If there is an option missing, file an issue for us.
+
+## Dry run
+
+If you want to perform, for any reason, a dry run where the main build command is `echo`ed to the console rather than executed, just set `IMAGE_BUILDER_DRY_RUN=1` environment variable when executing packer. Good for demos or testing the integration.
+
+## Building without Packer
+
+To test this library directly without packer, do:
+
+    go run github.com/lzap/packer-plugin-image-builder/cmd/ibpacker/ -help
+
+Use options to initiate a build:
+
+```
+Usage of ibpacker:
+  -arch string
+        architecture (default "x86_64")
+  -blueprint string
+        path to blueprint file
+  -distro string
+        distribution name (fedora, centos, rhel, ...) (default "fedora")
+  -dry-run
+        dry run
+  -hostname string
+        SSH hostname or IP with optional port (e.g. example.com:22)
+  -type string
+        image type (minimal-raw, qcow2, ...) (default "minimal-raw")
+  -username string
+        SSH username
+```
+
+For example:
+
+```
+git clone github.com/lzap/packer-plugin-image-builder
```

Comment:
```suggestion
git clone git@github.com:osbuild/packer-plugin-image-builder.git
```

Reply:
Going for `git clone https://github.com/osbuild/packer-plugin-image-builder`